sdk/Sdk.Generators/FunctionExecutor/FunctionExecutorGenerator.Emitter.cs

```diff
@@ -67,20 +69,40 @@ public static IHostBuilder ConfigureGeneratedFunctionExecutor(this IHostBuilder
                 return result;
             }
 
+            private static string EmitCreateDefaultExecutorMethod(GeneratorExecutionContext context)
+            {
+                var workerCoreAssembly = context.Compilation.SourceModule.ReferencedAssemblySymbols.Single(a => a.Name == "Microsoft.Azure.Functions.Worker.Core");
```

Comment:
Can some of the strings in this class be constants (in this file or another)?

Reply:
moved assembly name to a constant.